src/platforms/react-native/manual-setup/hermes.mdx

```diff
@@ -56,6 +64,12 @@ npx react-native bundle --platform android --dev false --entry-file index.js --r
 npx react-native bundle --platform ios --dev false --entry-file index.js --reset-cache --bundle-output main.jsbundle --sourcemap-output main.jsbundle.map --minify false
 ```
 
+<Note>
+
+If you are running Hermes with JS Bundle you only need step one, but replace `main.jsbundle.packager.map` and `index.android.bundle.packager.map` with `main.jsbundle.map` and `index.android.bundle.map` in the commands respectively.
+
+</Note>
```

Comment:
```suggestion
```

Does it make sense to combine this note with the above info on which steps are required based on your app bundle was generated?